src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1.cs

```diff
@@ -797,6 +805,9 @@ public ImmutableArray<T> RemoveRange(IEnumerable<T> items, IEqualityComparer<T>
             self.ThrowNullRefIfNotInitialized();
             Requires.NotNull(items, "items");
             Requires.NotNull(equalityComparer, "equalityComparer");
+            
+            if (self.IsEmpty)
+                return self;
```

Comment:
This change is also the cause of the test failure, I believe, as it also changes behavior, and there's a test that catches that.